cfme/networks/cloud_network.py

```diff
@@ -79,7 +79,7 @@ def delete(self):
         """Delete this cloud network"""
         view = navigate_to(self, 'Details')
         view.toolbar.configuration.item_select('Delete this Cloud Network', handle_alert=True)
-        view.flash.assert_success_message('The selected Cloud Network was deleted')
```

Comment:
**REQUIRED:** Please check if this is the case for 5.8 too

Reply:
Checked that same flash message is displayed in both, 5.8 and 5.9.